docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,79 +2,39 @@
  
----
-#### Detailed instructions for adding a new gene
-
-1. Find the gene in HGNC that you need to add:
-https://www.genenames.org/. Copy the ID (for example, 8965)
-2. Open the `src/ontology/imports/hgnc_terms.txt` file
-3. Add a new line to the file: http://identifiers.org/hgnc/[your ID], for example http://identifiers.org/hgnc/8965
-5. Run import:
-`sh run.sh make refresh-merged`
-4. Close Protege and open again
-5. Edit in Protege
-6. In your diff, you will see changes to src/ontology/imports/hgnc_terms.txt and src/ontology/imports/merged_import.owl
-7. Commmit changes on a branch and create a pull request.
-
-Note: if an import does not have a `src/ontology/imports/*_terms.txt` file (e.g. for adding NCIT terms). The term(s) to import should be added to src/ontology/imports/manual_seed.txt.  When updating the import, the term(s) will seed the import files, which are pulled in when refreshing modules.
 
-### Alternate instructions
 
-#### Add classes from external ontologies using a Text Editor
+### Alternate approaches
+These alternate approaches are mentioned here for documentation purposes. These should **only be used by experienced curators**, and followed by a clean-up of class declarations in mondo-edit.obo 
 
-1. Close Protege
-1. Checkout master
-1. Create a new branch
-1. Open mondo-edit.obo with a text editor like Sublime or TextEdit
-1. Add axiom in mondo-edit.obo *text file*
-1. **For example:**  relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/129
-1. Save text file
-1. open mondo-edit.obo in Protege
-1. File -> save as (obo format, save as mondo-edit.obo)
-1. Replace existing file  
-1. Check diff
-1. Commit/Push
+- **Add the required axiom, using classes from external ontologies, the in mondo-edit.obo *text file* using a Text Editor**
+ For example:  relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/129
 
-#### Add classes from external ontologies using Protege
 
-1. Select 'owl:Thing'
-1. Add a new class
-paste the full iri in the 'Name:' field, for example, http://purl.obolibrary.org/obo/CHEBI_50906.
-1. hit 'OK'
-1. Now you can use this term, for example to construct logical axioms. The next time the imports are refreshed, the metadata (labels, definitions, etc) for this term is imported from the respective external source ontology and becomes visible in mondo-edit.obo.
-1. See instructions on how to remake the imports file [here](../developer-guide/imports.md).
+- **Add classes from external ontologies using Protege.**
+These classes should be added as new classses under 'owl:Thing', using the full IRI in the 'Name:' field, for example, http://purl.obolibrary.org/obo/CHEBI_50906.
 
+While classes from external ontologies can be added directly using Protege, this process results in having class declarations in mondo-edit.obo and require further work to be cleaned out.
```

Comment:
Unfortunately, reading this line (I know I suggested) here in isolation from Nico's comment doesn't have the impact of don't do this or at least that's what I understood his comment as.

Take 2 "...directly using Protege, this process results in having < annoying - is there a better word, extraneous?> class declarations in mondo-edit.obo and require further manual work to remove these. Therefore, it's best to follow the Preferred Instructions rather than try to make these updates using Protege."

Reply:
the "alternate approaches" section was removed